selenium/che-selenium-test/pom.xml

```diff
@@ -106,11 +106,6 @@
             <artifactId>testng</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.eclipse.che.plugin</groupId>
```

Comment:
We need it at runtime to have GitHubKey DTO implementation

Reply:
Can you guys decide on which way is correct and make a change according to that?

Reply:
I didn't manage to reproduce the error regarding with these runtime dependencies. But if we get I'd remove these for now, and return if the problem occurs again

Reply:
Sounds good!